packages/EasyStandard/docs/ecs/sniffs.md

```diff
@@ -1,11 +1,39 @@
----eonx_docs---
-title: 'EasyStandard: Sniffs'
-weight: 1001
----eonx_docs---
-#### Classes
+---eonx_docs--- title: 'EasyStandard: Sniffs' weight: 1001 ---eonx_docs---
+
+#### Arrays
+
+##### [\EonX\EasyStandard\Sniffs\Arrays\OrderArrayKeysAlphabeticallySniff][1]
+
+Arrays must be sorted by keys alphabetically.
+
+```php
+// incorrect
+$array = ['z' => 1, 'g' => 2, 'a' => 3];
+```
+
+```php
+// correct
+$array = ['a' => 3, 'g' => 2, 'z' => 1];
+```
+
+**Parameters**
+
+- `skipPatterns` - List of templates, if they match, the check is ignored. The type of token for which you want to
```

Comment:
Please change the description:
```
A list of patterns to be checked to skip the array. Specify a token type (e.g. `T_FUNCTION` or `T_CLASS`) as a key and an array of regex patterns as a value to skip an array in the corresponding tokens (functions, classes). Default value: `[]`.
For example, you can skip all the arrays inside of functions which names start with `someFunction` or classes which names start with `SomeClass`.
```
And replace `/^someClass.*/` with `/^SomeClass.*/`.

Reply:
Done.